src/simplewallet/simplewallet.cpp

```diff
@@ -3149,40 +3149,40 @@ Pending or Failed: "failed"|"pending",  "out", Lock, Checkpointed, Time, Amount*
-                           tr(tools::wallet_rpc::LNS_BUY_MAPPING::description));
-
-  m_cmd_binder.set_handler("lns_renew_mapping",
-                           [this](const auto& x) { return lns_renew_mapping(x); },
-                           tr(USAGE_LNS_RENEW_MAPPING),
-                           tr(tools::wallet_rpc::LNS_RENEW_MAPPING::description));
-
-  m_cmd_binder.set_handler("lns_update_mapping",
-                           [this](const auto& x) { return lns_update_mapping(x); },
-                           tr(USAGE_LNS_UPDATE_MAPPING),
-                           tr(tools::wallet_rpc::LNS_UPDATE_MAPPING::description));
-
-  m_cmd_binder.set_handler("lns_encrypt",
-                           [this](const auto& x) { return lns_encrypt(x); },
-                           tr(USAGE_LNS_ENCRYPT),
-                           tr("Encrypts a LNS mapping value with a given name; primarily intended for use with external mapping update signing"));
-
-  m_cmd_binder.set_handler("lns_print_owners_to_names",
-                           [this](const auto& x) { return lns_print_owners_to_names(x); },
-                           tr(USAGE_LNS_PRINT_OWNERS_TO_NAMES),
+  m_cmd_binder.set_handler("ons_buy_mapping",
+                           [this](const auto& x) { return ons_buy_mapping(x); },
+                           tr(USAGE_ONS_BUY_MAPPING),
+                           tr(tools::wallet_rpc::ONS_BUY_MAPPING::description));
+
+  m_cmd_binder.set_handler("ons_renew_mapping",
+                           [this](const auto& x) { return ons_renew_mapping(x); },
+                           tr(USAGE_ONS_RENEW_MAPPING),
+                           tr(tools::wallet_rpc::ONS_RENEW_MAPPING::description));
+
+  m_cmd_binder.set_handler("ons_update_mapping",
+                           [this](const auto& x) { return ons_update_mapping(x); },
+                           tr(USAGE_ONS_UPDATE_MAPPING),
+                           tr(tools::wallet_rpc::ONS_UPDATE_MAPPING::description));
+
+  m_cmd_binder.set_handler("ons_encrypt",
+                           [this](const auto& x) { return ons_encrypt(x); },
+                           tr(USAGE_ONS_ENCRYPT),
+                           tr("Encrypts a ONS mapping value with a given name; primarily intended for use with external mapping update signing"));
+
+  m_cmd_binder.set_handler("ons_print_owners_to_names",
+                           [this](const auto& x) { return ons_print_owners_to_names(x); },
+                           tr(USAGE_ONS_PRINT_OWNERS_TO_NAMES),
                            tr("Query the Oxen Name Service names that the keys have purchased. If no keys are specified, it defaults to the current wallet."));
 
-  m_cmd_binder.set_handler("lns_print_name_to_owners",
-                           [this](const auto& x) { return lns_print_name_to_owners(x); },
-                           tr(USAGE_LNS_PRINT_NAME_TO_OWNERS),
+  m_cmd_binder.set_handler("ons_print_name_to_owners",
```

Comment:
And this one

Reply:
Maybe `ons_print_owners_to_names` -> `ons_by_owner`
and `ons_print_names_to_owners` -> `ons_lookup`?